airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/http_requester.py

```diff
@@ -547,6 +548,8 @@ def _try_get_error(value: Any) -> Any:
                     or value.get("errors")
                     or value.get("failures")
                     or value.get("failure")
+                    or value.get("details")
```

Comment:
let's add test cases for these values

Reply:
I've already added it [here](https://github.com/airbytehq/airbyte/pull/32545/files#diff-eb7a9110211a5b84c7039ade89d0b0385d480ba9192a41f7a280f7809b6fecefR689), or you mean some else tests?

Reply:
not sure how I missed it. looks good then!